### PR TITLE
Google Content API for Shopping Integration

### PR DESCRIPTION
**Pull request by panos-span**
## Description

Please include a summary of the change and the issue it solves. 

**Fixes** #5498

## Type of change

(Please delete options that are not relevant)

- [x] ⚡ New feature (non-breaking change which adds functionality)

### What is the solution?

I have created a class for the google content shopping handler and for the Accounts, Orders and Products API Table and the connection is achieved using the libraries `google-auth-oauthlib`, `google-auth-httplib2` and `google-api-python-client`.

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation, or created issues to update them.
- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).

### CODE REVIEW

File: mindsdb/integrations/handlers/google_content_shopping_handler/google_content_shopping_handler.py [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+            if os.path.exists('token_content.json'):
+                self.credentials = Credentials.from_authorized_user_file('token_content.json', self.scopes)
+            if not self.credentials or not self.credentials.valid:
+                if self.credentials and self.credentials.expired and self.credentials.refresh_token:
+                    self.credentials.refresh(Request())
+                else:
+                    self.credentials = service_account.Credentials.from_service_account_file(
+                        self.credentials_file, scopes=self.scopes)
+            # Save the credentials for the next run
+            with open('token_content.json', 'w') as token:
```
Comment (StpMax) [inline mindsdb/integrations/handlers/google_content_shopping_handler/google_content_shopping_handler.py:70]: Please, do not save/load files in current dir. Use `file_storage` instead:
```python
def __init__(self, name: str, **kwargs):
    self.fs_storage = kwargs['file_storage']
    
def connect(self):
    ...
    self.fs_storage.file_set('token_content.json',  '{"key": "value"}'.encode())   # second param must by bytes, therefore `.encode()` there
    json_str_bytes = self.fs_storage.file_get('token_content.json')    # will raise exception if file not exists
    json_str = json_str_bytes.decode()
```

File: mindsdb/integrations/handlers/google_content_shopping_handler/__init__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    import_error = e
+
+title = 'Google Content Shopping'
+name = 'google_content_shopping'
+type = HANDLER_TYPE.DATA
+icon_path = 'icon.png'
+
+__all__ = [
+    'Handler', 'version', 'name', 'type', 'title', 'description',
+    'import_error', 'icon_path'
```
Comment (StpMax) [inline mindsdb/integrations/handlers/google_content_shopping_handler/__init__.py:19]: If possible, please add also `connection_args` and `connection_args_example`

**Review by StpMax — CHANGES_REQUESTED** [review]

File: mindsdb/integrations/handlers/google_content_shopping_handler/google_content_shopping_handler.py (hunk 1 above)
Comment (panos-span) [reply mindsdb/integrations/handlers/google_content_shopping_handler/google_content_shopping_handler.py]: Thank you for your detailed input. Made the necessary changes.

File: mindsdb/integrations/handlers/google_content_shopping_handler/__init__.py (hunk 2 above)
Comment (panos-span) [reply mindsdb/integrations/handlers/google_content_shopping_handler/__init__.py]: Done.

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**